src/main/webapp/help/parasoftCoverage/help-criticality.html

```diff
@@ -1,18 +1,18 @@
 <div>
-    When a quality gate has been missed, this property determines whether the result of the associated coverage stage will
+    When a quality gate fails, this property determines whether the result of the associated coverage stage will
```

Comment:
I have a general feedback here:
@rwang-parasoft Are we using "unstable" and "failure" here following the original design of code coverage API?
I believe it's better to be consistent with Jenkins by using "Failed" instead of "Failure'.
<img width="154" alt="image" src="https://github.com/jenkinsci/parasoft-findings-plugin/assets/35082985/ddc6912e-ce35-445d-9608-eacce62f1ca6">
<img width="127" alt="image" src="https://github.com/jenkinsci/parasoft-findings-plugin/assets/35082985/5aad8003-6a24-4a7b-85ac-fa22d3fb6d38">

Reply:
Change it to: When a quality gate fails, this property determines whether the result of the associated coverage stage will be marked as unstable or failed.